Client.php

```diff
@@ -54,49 +54,70 @@ class Client extends Connection
      *
      * @const int
      */
-    const ASYNCHRONOUS      = STREAM_CLIENT_ASYNC_CONNECT;
+    const ASYNCHRONOUS        = STREAM_CLIENT_ASYNC_CONNECT;
 
     /**
      * Open client socket connection.
      *
      * @const int
      */
-    const CONNECT           = STREAM_CLIENT_CONNECT;
+    const CONNECT             = STREAM_CLIENT_CONNECT;
 
     /**
      * Client socket should remain persistent between page loads.
      *
      * @const int
      */
-    const PERSISTENT        = STREAM_CLIENT_PERSISTENT;
+    const PERSISTENT          = STREAM_CLIENT_PERSISTENT;
 
     /**
      * Encryption: SSLv2.
      *
      * @const int
      */
-    const ENCRYPTION_SSLv2  = STREAM_CRYPTO_METHOD_SSLv2_CLIENT;
+    const ENCRYPTION_SSLv2    = STREAM_CRYPTO_METHOD_SSLv2_CLIENT;
 
     /**
      * Encryption: SSLv3.
      *
      * @const int
      */
-    const ENCRYPTION_SSLv3  = STREAM_CRYPTO_METHOD_SSLv3_CLIENT;
+    const ENCRYPTION_SSLv3    = STREAM_CRYPTO_METHOD_SSLv3_CLIENT;
 
     /**
      * Encryption: SSLv2.3.
      *
      * @const int
      */
-    const ENCRYPTION_SSLv23 = STREAM_CRYPTO_METHOD_SSLv23_CLIENT;
+    const ENCRYPTION_SSLv23   = STREAM_CRYPTO_METHOD_SSLv23_CLIENT;
 
     /**
      * Encryption: TLS.
      *
      * @const int
      */
-    const ENCRYPTION_TLS    = STREAM_CRYPTO_METHOD_TLS_CLIENT;
+    const ENCRYPTION_TLS      = STREAM_CRYPTO_METHOD_TLS_CLIENT;
+
+    /**
+     * Encryption: TLSv1.0.
+     *
+     * @const int
+     */
+    const ENCRYPTION_TLS_v1_0 = STREAM_CRYPTO_METHOD_TLSv1_0_CLIENT;
```

Comment:
Yep I agree, I though that the version format was the same than the PHP constant `SSLv23` so I took the same scheme :smile:
I'll remove the first `_`, I think it's more readable ! I let SSL untouched.